package.json

```diff
@@ -17,11 +17,14 @@
     "deep-iterator": "^1.1.0",
     "deepmerge": "^2.2.1",
     "del": "^3.0.0",
+    "download": "^7.1.0",
     "eslint": "^5.16.0",
     "eslint-config-airbnb-base": "^13.1.0",
     "eslint-plugin-import": "^2.17.1",
     "fancy-log": "^1.3.3",
     "flat": "^4.1.0",
+    "font-ranger": "^3.0.5",
```

Comment:
This library is a bit obscure, but nothing else worked in the way we need. (They rely on reading a page to work out what characters are used, but we can't have that.)

It mostly delegates to `pyftsubset`, so I think it's ok.

Reply:
Seems a reasonable choice for our requirements.